src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -23,16 +35,53 @@ public HttpClientHandlerTest(ITestOutputHelper output)
         }
         
         [Fact]
-        public void SendAsync_SimpleGet_Success()
+        public async void SendAsync_SimpleGet_Success()
+        {
+            using (var handler = new HttpClientHandler())
+            {
+                using (var client = new HttpClient(handler))
+                {
+                    // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
+                    HttpResponseMessage response = await client.GetAsync(HttpTestServers.RemoteGetServer);
+                    Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    string responseContent = await response.Content.ReadAsStringAsync();
+                    _output.WriteLine(responseContent);
+                }
+            }
+        }
+
+        [Fact]
+        public async void SendAsync_SimpleHttpsGet_Success()
         {
-            var handler = new HttpClientHandler();
-            var client = new HttpClient(handler);
-            
-            // TODO: This is a placeholder until GitHub Issue #2383 gets resolved.
-            var response = client.GetAsync(HttpTestServers.RemoteGetServer).Result;
-            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-            var responseContent = response.Content.ReadAsStringAsync().Result;
-            _output.WriteLine(responseContent);
+            using (var handler = new HttpClientHandler())
+            {
+                using (var client = new HttpClient(handler))
+                {
+                    HttpResponseMessage response = await client.GetAsync(HttpTestServers.SecureRemoteGetServer);
+                    string responseContent = await response.Content.ReadAsStringAsync();
+		                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                    _output.WriteLine(responseContent);
```

Comment:
I was worried if adding further asserts would look like I am testing too many things in a single test. 
The validation on content can be added for other tests. Am more than happy to assert on the content (and any other) as well.